### ISSUE TITLE
Emoji with underscores aren't always displayed properly

### ISSUE THREAD

**Issue by jedcn** [jedcn/reveal-ck#29]
If you create markdown slides that look like this (minimal case to reproduce):

```markdown
:money_with_wings:
```

Then I'd expect to see a single slide with the :money_with_wings: emoji.

Instead, it's translated into:

```html
<p>:money<em>with</em>wings:</p>
```

I suspect this is happening because the markdown processor looks "into" the emoji when it should respect the boundaries of the emoji.

The same thing happens with emoji that have a single underscore but appear on the same line:

```markdown
:blue_heart: :+1: :blue_heart:
```

Should be :blue_heart: :+1: :blue_heart:, but instead the opening _ and closing _ become `<em>`s:

```html
<p>:blue<em>heart: ... :blue</em>heart:</p>
```

**Comment by jedcn** [comment]
Yeah. reveal-ck's markdown processor was finding the `_` and presuming it was the beginning of _emphasis_.

**Comment by jedcn** [comment]
This was fixed in release 0.6.1.
